setup.py

```diff
@@ -41,6 +41,7 @@ def read(fname):
 	'pandas',
 	'sklearn',
 	'keras',
+	'torch',
```

Comment:
I'd like to push some deeplearning packages to `extras_require`,  some traditional machine learning algorithm may not require them.

Reply:
Yes, good point